optimizely/config/datafileprojectconfig/entities/entities.go

```diff
@@ -67,10 +67,16 @@ type trafficAllocation struct {
 
 // Variation represents an experiment variation from the Optimizely datafile
 type Variation struct {
-	ID string `json:"id"`
-	// @TODO(mng): include variables
-	Key            string `json:"key"`
-	FeatureEnabled bool   `json:"featureEnabled"`
+	ID             string              `json:"id"`
+	Variables      []VariationVariable `json:"variables"`
```

Comment:
I think we can use a map here to make look-ups easier.

Reply:
I have updated the public facing dataModel with a map.
https://github.com/optimizely/go-sdk/pull/71/files#diff-f55225b09df17f6c9183261f39659ee1R22